focus-blocked/js/main.js

```diff
@@ -150,12 +150,14 @@ try {
   if (cuddly_bear_mode && !strict_blocking) {
     cuddlyBearBtn.className = 'showCuddlyBearBtn';
     unblockBtn.addEventListener('click', () => {
-      window.open(
-        `${current_url}${
-          current_url.includes('?') ? '&' : '?'
-        }focus_bear_temporarily_allow=true&unblock_url=true`,
-        '_self'
-      );
+      //@Description: desktop apps requested and the rest of operation will be handle by the desktop apps
```

Comment:
I think we'll need to have a version check otherwise we'll break it for old versions. @manishjain999 can you add an extra URL parameter if the desktop app is ready to support the new approach?

Reply:
Yes sure, we can add version=1.13.301 for mac and same for Windows.
The block page should do the new approach if the version param is found, if the "version" param is not found, keep the old approach.

Reply:
![image](https://github.com/user-attachments/assets/c5ad6738-c7be-44be-8880-f9f8fa4be54c)